angular/modules/README.md

```diff
@@ -0,0 +1,32 @@
+# Module
+
+Grouping components into a module. Learn more [https://angular.io/guide/architecture-modules](https://angular.io/guide/architecture-modules)
+
+## When to use module?
+
+- Each menu view should has its own module.
+- When you have a component that depend on multiple components.
+- Component, Pipe that will be used in multiple modules.
+
+## Why module?
+
+- Angular is not allow you to declare a component in multiple modules, but you can import a module into multiple modules.
+- Module is very reusable, and easy to integrate. You would only need to import a module and reuse multiple components that exported.
+
+## How to create module?
```

Comment:
I attached the URL for module learn more: https://angular.io/guide/architecture-modules#ngmodule-metadata I think we should based it on the document instead since it could be change in the future. We should only recommend the best practices, framework feature should already explain in the official document.